docs/source/recipes/qat_distributed.rst

```diff
@@ -0,0 +1,88 @@
+.. _qat_distributed_recipe_label:
+
+=============================================
+Distributed Quantization-Aware Training (QAT)
+=============================================
+
+QAT allows for taking advantage of memory-saving optimizations from quantization at inference time, without significantly
+degrading model performance. In torchtune, we use `torchao <https://github.com/pytorch/ao>`_ to implement QAT during training.
+This works by :ref:`simulating quantization numerics during fine-tuning <what_is_qat_label>`. While this may introduce memory and
+compute overheads during training, our tests found that QAT significantly reduced performance degradation in evaluations of
+quantized model, without compromising on model size reduction gains.
+
+Interested in using this recipe? Check out some of our tutorials which show how it is used:
+
+* :ref:`qat_finetune_label`
+
+The best way to get started with our recipes is through the :ref:`cli_label`, which allows you to
+list all our recipes and configs, run recipes, copy configs and recipes, and validate configs
+without touching a line of code!
+
+For example, if you're interested in using this recipe with the latest `Llama models <https://llama.meta.com/>`_, you can fine-tune
+in just two steps:
+
+.. note::
+
+    You may need to be granted access to the Llama model you're interested in. See
+    :ref:`here <download_llama_label>` for details on accessing gated repositories.
+
+
+.. code-block:: bash
+
+    tune download meta-llama/Meta-Llama-3-8B-Instruct  \
+    --output-dir /tmp/Meta-Llama-3-8B-Instruct \
+    --ignore-patterns "original/consolidated.00.pth" \
+    --HF_TOKEN <HF_TOKEN>
+
+    tune run --nproc_per_node 4 qat_distributed \
```

Comment:
If it requires at least 6 GPUs we probably should set a higher value of nproc_per_node 😃

Reply:
good catch : )